master.sh

```diff
@@ -40,13 +40,7 @@ init() {
 	fi
 
 	# Set working mode
-	if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
-		echo "Start k8s cluster in installation mode"
 		RESTART_POLICY="always"
```

Comment:
currently, we still have a few container outside of kublet: etcd, flannel, proxy.

We change the start mode to the use case accordingly .

Of course the goal would be to move them into kublet